syncmaster/worker/celery.py

```diff
@@ -0,0 +1,6 @@
+# SPDX-FileCopyrightText: 2023-2024 MTS PJSC
+# SPDX-License-Identifier: Apache-2.0
+from syncmaster.worker import celery_factory
+from syncmaster.worker.settings import WorkerAppSettings
+
+app = celery_factory(WorkerAppSettings())
```

Comment:
Shouldn't we update `python -m celery -A syncmaster.worker.celery` entrypoint & usage examples to use a new location?

Reply:
The path remains the same, Celery supports it https://docs.celeryq.dev/en/stable/getting-started/next-steps.html#about-the-app-argument